recipes/newrelic/infrastructure/ohi/apache/debian.yml

```diff
@@ -40,8 +40,14 @@ inputVars:
     prompt: "Apache status URL (default: http://127.0.0.1/server-status?auto)"
     default: "http://127.0.0.1/server-status?auto"
 
-install:
+preInstall:
+  info: |2
+      To capture data from the Apache integration, you'll first need to meet these prerequisites:
+      - Apache versions 2.2 or 2.4
```

Comment:
I'm not sure we should say that. If later on the ohi support a more recent version this is going to become inaccurate. We should remove this line, or update it with a link to our doc to refer to the supported version